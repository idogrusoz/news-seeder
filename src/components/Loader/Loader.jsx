import React from 'react';
import { BounceLoader } from 'react-spinners';

const Loader = () => {
    const styles = {
        wrapper: {
            width: '100%',
            height: '100%',
            display: 'flex',
            justifyContent: 'center',
            alignItems: 'center',
        },
    };
    return (
        <div style={styles.wrapper}>
            <BounceLoader color="#37e8b8" />
        </div>
    );
};

export default Loader;
