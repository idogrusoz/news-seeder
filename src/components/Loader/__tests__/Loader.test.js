import { shallow } from 'enzyme';
import Loader from '../Loader.jsx';

describe('Loader test', () => {
    it('Matches the snapshot', () => {
        const wrapper = shallow(<Loader />);
        expect(wrapper).toMatchSnapshot();
    });
});
