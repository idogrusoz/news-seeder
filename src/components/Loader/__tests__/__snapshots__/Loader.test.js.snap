// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader test Matches the snapshot 1`] = `ShallowWrapper {}`;
